events/wc_offmap_events.txt

```diff
@@ -958,4 +958,326 @@ character_event = {
 			}
 		}
 	}
+}
+character_event = {
+	id = WCOME.500
+	picture = GFX_evt_old_god_healing
+	desc = EVTDESC_WCOME.500
+	border = GFX_event_normal_frame_religion
+	
+	is_triggered_only = yes
+	
+	only_playable = yes
+	
+	option = {
+		name = EVTOPTA_WCOME.500
+		trigger = {
+			OR = {
+				has_injury_trigger = yes
+				trait = maimed
+				trait = mangled
```

Comment:
Maimed and mangled are maimed traits, what are they doing here?

Reply:
i dont get what you mean

Reply:
> i dont get what you mean

The maimed traits (one-eyed, one_handed, one_legged, disfigured) are removed in the last option of the event, so I don't get why these 2 are here.